### PR TITLE
ENH: addition of the argument show=True in plot_psd()

### PR DESCRIPTION
**Pull request by cmoutard**
Many plot_XXX functions do have the show(=True) argument contrary to plot_psd().
I needed to use it, so here is this small PR adding this arg in this function.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
+1 from me for completeness, I'm not worried about it not being at the end since it's so far into a long list of kwags. Okay with you @agramfort?

**Comment by Eric89GXL** [comment]
If this manages to break someone's code because they used enough `args` to have it not work anymore, we should award them with some sort of prize.

**Comment by agramfort** [comment]
LGTM 

+1 for merge when you make travis happy. pyflakes error

File: mne/io/base.py [hunk 1]
```diff
@@ -1139,8 +1141,8 @@ def plot_psd(self, tmin=0.0, tmax=60.0, fmin=0, fmax=np.inf,
         return plot_raw_psd(self, tmin=tmin, tmax=tmax, fmin=fmin, fmax=fmax,
                             proj=proj, n_fft=n_fft, picks=picks, ax=ax,
                             color=color, area_mode=area_mode,
-                            area_alpha=area_alpha,
-                            n_overlap=n_overlap, dB=dB, n_jobs=n_jobs)
+                            area_alpha=area_alpha, n_overlap=n_overlap,
```
Comment (dengemann) [inline mne/io/base.py]: looks like you're having a whitespace here at the end of the line. --> pep8

**Comment by dengemann** [comment]
+1 for merge once travis is happy

**Comment by cmoutard** [comment]
I don't know what happened here, I never closed it. Unfortunately, I won't have time to clean this. Maybe this PR should just be ignored ?

**Comment by Eric89GXL** [comment]
It auto-closed -- you must have push forced a branch here that had no new commits relative to upstream master. Feel free to open a new PR if you come back to this at some point.

**Comment by mainakjas** [comment]
If you messed up, you can use `git reflog` to recover. Google it :)

**Comment by cmoutard** [comment]
Thanks @mainakjas !!
I am opening a new PR on @Eric89GXL 's advice.

**Pull request closed without merging**